flytekit/clis/flyte_cli/main.py

```diff
@@ -511,12 +518,50 @@ def _flyte_cli(ctx, project, domain, name, host, insecure):
     pass
 
 
+########################################################################################################################
+#
+#  Flyte-cli auth config set-up options.
+#
+########################################################################################################################
+
+_client_id_option = _click.option(
+    "--client-id",
+    required=False,
+    type=str,
+    default=None,
+    help="""This is the public identifier for the app which handles authorization for a Flyte deployment.
+            More details here: https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/."""
+)
+
+_redirect_uri_option = _click.option(
+    "--redirect-uri",
+    required=False,
+    type=str,
+    default=None,
```

Comment:
This needs to be set.  we should pick something.  I like `http://localhost:53593/callback`